auto_landing/CMakeLists.txt

```diff
@@ -8,11 +8,10 @@ add_definitions(-std=c++17)
 set(CMAKE_BUILD_TYPE Release)
 
 
-#cs_add_library(auto_landing_lib src/template_ros.cpp src/template_ros_pub.cpp)
-#target_link_libraries(auto_landing_lib ${PCL_LIBRARIES})
+cs_add_library(auto_landing_lib src/tracking.cpp)
```

Comment:
instead of putting auto_landing again and again use ${PROJECT_NAME} ... applies for complete CMake